### ISSUE TITLE
An “img” element must have an “alt” attribute, except under certain conditions

### ISSUE THREAD

**Issue by pdehaan** [mozilla/send#440]
via #139 

```
line 47   col 26  Error  An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.  html-checker
line 109  col 3   Error  An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.  html-checker
line 116  col 57  Error  An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.  html-checker
line 124  col 91  Error  An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.  html-checker
line 125  col 91  Error  An “img” element must have an “alt” attribute, except under certain conditions. For details, consult guidance on providing text alternatives for images.  html-checker
```

Possibly related/duplicated:

```
line 112  col 57  Error  Images must have alternate text                        axe
line 120  col 91  Error  Images must have alternate text                        axe
line 121  col 91  Error  Images must have alternate text                        axe
```

This may be a bit trickier to debug, since we're setting some localized `alt=""` tags via l20n.

**Comment by urohit011** [comment]
I would like to work on this.

**Comment by pdehaan** [comment]
@urohit011 That would be awesome, thanks. 👍
Let us know if you have any questions.

**Comment by urohit011** [comment]
Can you please point me to one of the files that has to be fixed.

**Comment by varghesethomase** [comment]
You can find the images in main.handlebars. The images that does not have alt attribute are located at the bottom fo the page ie mozilla logo to the left and twitter and github logo on the right @urohit011

**Comment by urohit011** [comment]
Thank you @varghesethomase

**Comment by dannycoates** [comment]
fixed in #474
